src/openpersonen/features/country_code_and_omschrijving/management/commands/import_country_codes_and_omschrijving.py

```diff
@@ -0,0 +1,62 @@
+import csv
+
+from django.core.management import BaseCommand
+
+import requests
+
+from openpersonen.features.country_code_and_omschrijving.models import (
+    CountryCodeAndOmschrijving,
+)
+
+
+class Command(BaseCommand):
+    """
+    Run using
+    python src/manage.py import_country_codes_and_omschrijving --url='https://url.com/file.csv'
+    or
+    python src/manage.py import_country_codes_and_omschrijving --file=/path/To/file.csv
+    """
+
+    help = "Read in a csv file from a url or file path and populate models"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--url", help="Url to csv file")
+        parser.add_argument(
+            "--file", help="The csv file containing the data to import."
+        )
+
+    def handle(self, **options):
+
+        if options.get("url") and options.get("file"):
+            self.stderr.write("Please specify a url or file, not both")
+            return exit(1)
```

Comment:
This could probably be combined by raising a `CommandError` (https://docs.djangoproject.com/en/3.1/howto/custom-management-commands/#django.core.management.CommandError)

```suggestion
            raise CommandError("Please specify a url or file, not both")
```